source_lists/wesnoth

```diff
@@ -370,7 +371,6 @@ team.cpp
 teambuilder.cpp
 terrain/filter.cpp
 tod_manager.cpp
-units/abilities.cpp
```

Comment:
as said the commit message. this is to make travis fail faster if a do something wrong during the pr (i don't build it locally)